### ISSUE TITLE
Reduce resolver matching conflicts on 'create' and 'run'

### ISSUE THREAD

**Issue by moul** [scaleway/scaleway-cli#212]
Actually:
- `scw run ubuntu-trusty` will looks for public images and user images, it's kind of typical to create a **snapshot** of a public ubuntu and name it **"ubuntu trusty with my stuff"**

I think we should try to get the same behavior for everyone using `scw run ubuntu-trusty` -> use the public image.
Then we can enforce user to specify a _prefix_ for user **images**, i.e:
- `scw run ubuntu-trusty` will only looks for public images with a name matching `ubuntu-trusty`
- `scw run user/ubuntu-trusty` will only looks for user images

I restricted the perimeter of this optimization to `create` and `run`, because some calls as `inspect` should see all the items matching a name I think, i.e:

---
- `scw inspect ubuntu-trusty` will looks for public images matching **ubuntu-trusty**, **user images** matching **ubuntu-trusty** but also **public bootscripts**, **user snapshots**, **user volumes** and most important: **user servers**

In summary:
- Add a `user/` prefix which will restrict everywhere the resolver to user contents (not only images)
- No prefix means no restriction
- Except for `create` and `run`: no prefix means only public images first

@aimxhaisse @QuentinPerez what do you think ?

**Comment by QuentinBrosse** [comment]
This issue has been automatically closed because it has not had recent activity. Feel free to reopen it if you think that it is necessary.
Note that we are working on a totally new version of the CLI to support all the new coming products.
Thank you for your contributions and stay tuned! :wink:
